ghostTemplate.yaml

```diff
@@ -0,0 +1,377 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  ghostFS:
+    Type: AWS::EFS::FileSystem
+    Properties:
+      PerformanceMode: generalPurpose
+      Encrypted: false
+  ghostMount0:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 0 
+        - !Ref DefaultVPCSubnets
+  ghostMount1:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 1 
+        - !Ref DefaultVPCSubnets
+  ghostMount2:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 2 
+        - !Ref DefaultVPCSubnets
+  ghostMount3:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 3 
+        - !Ref DefaultVPCSubnets
+  ghostAPI:
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      ContainerDefinitions:
+        - Image: registry.hub.docker.com/library/ghost
+          PortMappings:
+            - ContainerPort: 2368
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group: !Ref ghostAPILogGroup
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix: logs
+          MountPoints:
+            - 
+              ContainerPath: /var/lib/ghost/content
+              SourceVolume: ghostVolume
+          Name: 0
+          Environment:
+            - Name: url
+              Value: !Join [ '', [ 'http://', !GetAtt Loadbalancer.DNSName ] ]
+            # - Name: database__client
+            #   Value: sqlite3
+            # - Name: database__connection__filename
+            #   Value: '/var/lib/ghost/content/data/ghost-efs.db'
```

Comment:
Should this be cleaned up now?

Reply:
I've refactored all of this to sit on a conditional so that when EnvironmentTagName != 'prod', sqlite will be used instead of Aurora MySQL. I figured that's an easy no-fuss switch for users that will try to do the right thing based on context, but easy enough for users to refactor if they'd like to customize the condition.